app/controllers/api/praise_controller.rb

```diff
@@ -8,9 +8,12 @@ class Api::PraiseController < ApplicationController
   layout false
 
   def create
-    # Is this a valid Slack request?
-    if invalid_signature!
-      return
+    # Skip validation in tests
+    if Rails.env.test? === false
```

Comment:
if `test?` is a boolean value can we drop `=== false` and switch the `if` to an `unless`?